dcpy/utils/s3.py

```diff
@@ -38,7 +38,10 @@ def string_as_acl(s: str) -> ACL:
 
 
 def _make_folder(s: str):
```

Comment:
This function is used by `list_objects()`. The previous logic for this function gave an error if input was an empty string `""`. We would want to be able to use an empty string as input to list **all** objects in a S3 bucket.

Reply:
Good catch! However, would `s.rstrip("/") + "/"` accomplish the same thing?

Also, looking at this now, I would sort of expect `s3._make_folder()` to actually make a folder on s3. Do you have thoughts about a potentially more accurate fn name?

Reply:
@alexrichey - No, because for some reason, when I tried listing objects at the root of a bucket (all objects) with a prefix `"/"`, it returns nothing; prefix `""` returns all objects in a bucket. 

In terms of the name, I thought same, too. I think the idea behind the function (@fvankrieken - feel free to correct me) is to convert a given prefix into a directory. Not sure what fn name I would pick for that purpose.

Reply:
Note for myself: idea - `folderize`